### PR TITLE
Update README.md with Phase 2 updates

### PR DESCRIPTION
**Pull request by omerfirmak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,13 +129,35 @@ docker logs -f juno
...
+* [ ] Synchronisation: implement verification of state from layer 1.
+* [ ] JSON-RPC API [v0.3.0](https://github.com/starkware-libs/starknet-specs/tree/v0.3.0-rc1):
+    * [ ] Implement the remaining endpoints:
+        * [ ] `starknet_getStorageAt`
+        * [ ] `starknet_getClass`
+        * [ ] `starknet_getClassHashAt`
+        * [ ] `starknet_getClassAt`
+        * [ ] `starknet_syncing`
+        * [ ] `starknet_pendingTransactions` (this will not be implemented since we don’t plan to implement `pending` tag)
+
```
Comment (jelilat) [inline README.md:146]: We should add `starknet_getNonce` as well

Comment (omerfirmak) [reply README.md:147]: done

File: README.md [hunk 2]
```diff
@@ -129,13 +129,36 @@ docker logs -f juno
...
+    * [ ] Implement the remaining endpoints:
+        * [ ] `starknet_getStorageAt`
+        * [ ] `starknet_getClass`
+        * [ ] `starknet_getClassHashAt`
+        * [ ] `starknet_getClassAt`
+        * [ ] `starknet_getNonce`
+        * [ ] `starknet_syncing`
+        * [ ] `starknet_pendingTransactions` (this will not be implemented since we don’t plan to implement `pending` tag)
+        
+        Requires Pagination
```
Comment (jelilat) [inline README.md:148]: Not sure. But I think this is supposed to be an internal note

File: README.md [hunk 3]
```diff
@@ -129,13 +129,36 @@ docker logs -f juno
...
+        
+        * [ ] `starknet_getEvents`
+        
+        Requires Integration of [Starknet in Rust](https://github.com/lambdaclass/starknet_in_rust):
```
Comment (jelilat) [inline README.md:152]: Same as this

**Review by jelilat — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (omerfirmak) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -129,13 +129,31 @@ docker logs -f juno
...
-* [ ] Integrate cairo vm
-* [ ] Verification
-    * [ ] L1 verification
-    * [ ] Execution of all transactions from feeder gateway
-* [ ] Full RPC (according to 0.11.0)
-* [ ] Start p2p discussions
-* [ ] Infura and Alchemy integrations
+The focus of Phase 2 will be to Verify the state from layer 1 and implement the remaining JSON-RPC endpoints.
+
+* [ ] Starknet 0.11 support
```
Comment (IronGauntlets) [inline README.md:134]:
```suggestion
* [ ] Starknet v0.11.0 support
```

**Pull request merged**